Core/automation/lib/python/core/utils.py

```diff
@@ -5,6 +5,7 @@
 __all__ = [
     "validate_channel_uid",
     "validate_uid",
+    "get_item_value",
```

Comment:
Move this to the end of list, and the function to the end of the file.

Reply:
OK, I'm guessing this is a convention/practice I'm not familiar with. I did find the ordering of the functions baffling. If they're just in whatever order they were added, that makes sense of it.

Reply:
Maybe it's just me, but I think it makes the most sense to have them in the order added.